spec/index.html

```diff
@@ -351,6 +352,64 @@ <h3>Blank nodes</h3>
     </figure>
   </section>
 
+  <section id="section-triple-terms">
+    <h3>Triple terms</h3>
+
+    <p>As the basic unit of description in RDF, the triple is a simple
+    abstraction for stating a directed relationship between two resources.
+    Sometimes, we need to describe more detailed circumstances underlying such
+    relationships. While we could describe these using more sophisticated
+    modelling, there is a tension between simple, direct statements and more
+    complex, granular descriptions. This may require a trade-off between
+    directness and completeness.</p>
+
```

Comment:
Instead of going into details, how about following the above paragraph directly with an example:

_For instance, given the following:_
<pre class="example nohighlight">
&lt;Bob&gt; &lt;is interested in> &lt;the Mona Lisa&gt;
</pre>

_We may want to say the following about the relationship:_
<pre class="example nohighlight">
&lt;Bob&gt; &lt;is interested in&gt; &lt;the Mona Lisa&gt; 
    &lt;is an> &lt;Interest>
    &lt;4th of October 1998>
</pre>

Reply:
Going into details is what this is about, but I appreciate your advice to approach it gently. However, your suggested form can be misleading, as it appears to conflate the proposition with a circumstance thereof.

Reply:
I feel like `<Bob> <is interested in> <the Mona Lisa>` is already quite specific - it does not reflect a need for an occurrence / token. (Avoiding the discussion on whether occurrences are _always_ needed.) Hence, I didn't see an issue with this particular example. 

E.g., `<the Mona Lisa> <depicts> <cypress tree>` could be found in many different sources and at different dates.

Reply:
I think in this PR we should focus on _how_ to use reifiers, and continue the debate on _why_ they are needed in https://github.com/w3c/rdf-star-wg/issues/138? The primer should convey the design, but not all of the theory. (Perhaps RDF Concepts needs to cover more of that? Or Semantics; after all, it is where the original reification documentation resides.)

Reply:
I would actually prefer a graphical example, building on the figure we already have.
Below is a quick and dirty experiment

![image](https://github.com/user-attachments/assets/e37169db-59d3-474a-ba34-3c80cccce224)

Note that I just had the idea of linking the arc to the reifier with a wiggly line, which is reminiscent of the tilda we use in Turtle :-)

Reply:
Note that this graphical example would change the description of the example.
Here is an attempt to reuse @niklasl 's narrative about examples 5-7, adapting it to this figure instead.

Conceptually, we can *annotate* a simple statement with a reference to something more detailed.

[figure]

The dotted circle above the "is intersted in" arc is called a [reifier](#todo-link) and expresses a concrete circumstance of Bob's interest, with a type and a date of origin.

Formally, this annotation is composed of four triples: 

```
<Bob> <is interested in> <the Mona Lisa> .
<reifier> <is a concretization of> «( <Bob> <is interested in> <the Mona Lisa> )» .
<reifier> <is an> <Interest> .
<reifier> <since> <4th of October 1998> .
```

The object of the second triple is a [triple term](#todo-link) describing the statement of which the reifier is a concretization. Triple terms can only appear in the object position, and should only be used as the object of a special predicate representing this "concretization" relation.

Reply:
> I think in this PR we should focus on how to use reifiers, and continue the debate on why they are needed in https://github.com/w3c/rdf-star-wg/issues/138?

@niklasl hence my proposal of simply switching to

`<the Mona Lisa> <depicts> <cypress tree>`

Reply:
> Triple terms can only appear in the object position, and should only be used as the object of a special predicate representing this "concretization" relation.

Depending on the outcome of the discussion, this statement can be weakened a bit

Reply:
>Conceptually, we can annotate a simple statement with a reference to something more detailed.

> [figure]

> The dotted circle above the "is intersted in" arc is called a [reifier](https://github.com/w3c/rdf-primer/pull/16/files/b54250e8bd6679f03107fb18bdfd64e900ee0a6f#todo-link) and expresses a concrete circumstance of Bob's interest, with a type and a date of origin.

I feel there is a non sequitur here; the terms annotate, reference and details are no longer used in the second part.

A suggestion:

Conceptually, we can annotate a simple statement with something more detailed.
[figure]

The dotted circle annotates the "is interested in" arc and is called a [reifier](https://github.com/w3c/rdf-primer/pull/16/files/b54250e8bd6679f03107fb18bdfd64e900ee0a6f#todo-link).
A reifier expresses a concrete circumstance of Bob's interest, and details it with a type and a date of origin.

Reply:
Thank you @william-vw and @pchampin for these suggestions! I've tried to work much of that into [051a328](https://github.com/w3c/rdf-primer/pull/16/commits/051a328ed372464eb67dda95657cc44672638ffc).

Reply:
@TallTed:
> I have a feeling that confusion will increase due to `a <Interest>` and `<is interested in>`. Changing the latter to `<hasInterest>` might be enough to solve this.

The `<is interested in>` is the more abstract binary relationship, following the existing example in the primer. Changing it to `<hasInterest>` implies remodelling, to instead link from `<Bob>` to the reified `<Interest>`. That would be better modelling if such interests as resources where of primary interest (heh). But the point here is to show how such more detailed interests can link to the simple proposition itself, as a reification thereof. Thus allowing us to keep the (arguably too) simple relationship.

Reply:
@niklasl — My concerns about `<Interest>` and `<is interested in>` are more about `interest` being both a verb and a noun. Better for examples to use less ambiguous English.